content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -1,134 +1,112 @@
 ---
 reviewers:
 - erictune
-title: Init Containers
+title: Init containers
 content_template: templates/concept
 weight: 40
 ---
 
 {{% capture overview %}}
-This page provides an overview of Init Containers, which are specialized
-Containers that run before app Containers and can contain utilities or setup
-scripts not present in an app image.
-{{% /capture %}}
 
+{{< feature-state for_k8s_version="v1.6" state="stable" >}}
+
+This page provides an overview of init containers: specialized containers that run before
+the app containers in a {{< glossary_tooltip text="Pod" term_id="pod" >}}.
+Init containers can contain utilities or setup scripts not present in an app image.
+
+{{% /capture %}}
 
-This feature has exited beta in 1.6. Init Containers can be specified in the PodSpec
-alongside the app `containers` array. The beta annotation value will still be respected
-and overrides the PodSpec field value, however, they are deprecated in 1.6 and 1.7.
-In 1.8, the annotations are no longer supported and must be converted to the PodSpec field.
 
 {{% capture body %}}
-## Understanding Init Containers
 
-A [Pod](/docs/concepts/workloads/pods/pod-overview/) can have multiple Containers running
-apps within it, but it can also have one or more Init Containers, which are run
-before the app Containers are started.
 
-Init Containers are exactly like regular Containers, except:
+## Understanding init containers
+A Pod can have multiple containers running apps within it, but it can also have one or more
+init containers that are run before the app containers are started.
+
+Init containers are exactly like regular containers, except:
 
 * They always run to completion.
 * Each one must complete successfully before the next one is started.
 
-If an Init Container fails for a Pod, Kubernetes restarts the Pod repeatedly until the Init
-Container succeeds. However, if the Pod has a `restartPolicy` of Never, it is not restarted.
+If an init container fails for a Pod, Kubernetes restarts the Pod repeatedly until the init
+container succeeds. However, if the Pod has a `restartPolicy` of Never, it is not restarted.
 
-To specify a Container as an Init Container, add the `initContainers` field on the PodSpec as
-a JSON array of objects of type
-[Container](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#container-v1-core)
+To specify a container as an init container, add the `initContainers` field on the PodSpec as
+an array of objects of type
+[container](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#container-v1-core)
```

Comment:
When we are talking about API resources, we use the exact resource name. In this case, we use `Container`.

Reply:
Revised